docs/eos-learn/blockchain-security.md

```diff
@@ -13,4 +13,10 @@ Being a relatively new technology, in full swing of research and development, it
 
 In this talk we will talk about the main security problems that affect blockchains, especially with a focus on smart contracts. We will show how these flaws can be exploited and we will give some practical examples. We will also talk about some famous hacks in which attackers have been able to steal millions of dollars from some blockchain-based systems. Finally we will explain how these failures can be avoided. We will especially focus on EOSIO technology.
 
-<iframe width="100%" height="350" src="https://www.youtube.com/embed/tss1d0sow0o" frameborder="0" allow="true" allowfullscreen></iframe> 
\ No newline at end of file
+
+<iframe width="100%" height="350" src="https://www.youtube.com/embed/tss1d0sow0o" frameBorder="0" allow="true" allowFullScreen></iframe> 
+
+
+As mentioned above, no system is 100% secure. Therefore, it is necessary to take into account that attackers can steal information in one way or another, so it is important to know how to avoid these attacks and learn from them to improve the security of our systems and thus avoid future threats.
+
+So we have experienced first-hand some attacks in which attackers have been able to steal information from some systems, such as the attack on **(EOS SX Vault)**. In this case, the attacker found some vulnerabilities in the smart contract running on the blockchain. For this attack, a complete follow-up was given to analyze and learn from the vulnerability, you can follow up on the complete analysis of this attack at **[Analysis of the Vulnerability Found in the vaults.sx Smart Contract](https://eoscostarica.medium.com/analysis-of-the-vulnerability-found-in-the-vaults-sx-smart-contract-445c8c968b5f)**.
```

Comment:
We have witnessed some attacks first-hand in which attackers have been able to steal information over even user funds, such as....

For this attack our team conducted an investigation to analyze and learn ...

Reply:
Ready